<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SharedMemoryInfoWidget</class>
 <widget class="QTabWidget" name="SharedMemoryInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="tabPosition">
   <enum>QTabWidget::South</enum>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>内存信息</string>
   </attribute>
   <widget class="QTableWidget" name="memoryHexInfoTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>500</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>291</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="memoryIDEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>内存ID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QComboBox" name="bufferTypeBox">
       <item>
        <property name="text">
         <string>循环缓冲区</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>覆盖缓冲区</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="viewButton">
       <property name="text">
        <string>查看</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="autoUpdateButton">
       <property name="text">
        <string>开始刷新</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>641</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>BufferSize</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="bufferSizeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>WritePointer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="writePointerEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>StationID</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QLineEdit" name="stationIDEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>BlockSize</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="blockSizeEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>ReadWriteLock</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="readWriteLockEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>IPAddress</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="ipAddressEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="memoryCharInfoTable">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>460</width>
      <height>350</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
